lib/manageiq/api/engine.rb

```diff
@@ -34,6 +34,10 @@ def self.log_kv(key, val)
       def vmdb_plugin?
         true
       end
+
+      def self.plugin_name
+        _('API')
```

Comment:
Should this be more specific like `REST API`?

Reply:
Sure, why not. Fixed.